.travis.yml

```diff
@@ -1,32 +1,52 @@
-sudo: false
+sudo: required
 language: erlang
 notifications:
   email: false
 
 otp_release:
```

Comment:
This is good. However, what do you think about having the lastest minor versions `(xx.3)` for each major OTP release?